app/assets/stylesheets/_honeycrisp-compact.scss

```diff
@@ -13,6 +13,13 @@
     min-height: 18rem;
   }
 
+  ul {
```

Comment:
i dont know why we remove the default list styling for ul/ol in honeycrisp. It's very odd and feels like an anti-pattern, as I'd much rather _remove_ bullet styling from a list that doesnt need it when we don't need it than globally not have it. 


For our honeycrisp compact overrides, I turned the list styles back on. Having them off would mean I'd need to target the ActionText editor's ul/ol to style them (because our global removal affects that component as well).

This means that there may be some instances of ul in the hub that have some bullets next to them now, but i didn't see any in my (cursory) look. ill take another pass and see what the client page looks like.